docs/CONTRIBUTORS.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html>
+<title>Google Offline Navigator: An open source project built for Grow with Google </title>
+
+<xmp theme="cerulean" style="display:none;" toc>
+# Contributors
+
+### Special thanks for all the people who had helped this project so far:
+
+<p class="txt-cntr">.</p> | <p class="txt-cntr">N</p> | <p class="txt-cntr">A</p>| <p class="txt-cntr">V</p>| <p class="txt-cntr">I</p> | <p class="txt-cntr">.</p>
+---------------- | ---------------- | ----------------- |---------------- | ----------------- | ----------------
+<p class="txt-cntr">![Brayan](https://avatars0.githubusercontent.com/u/17731807?s=60&v=1) [Bryan](https://github.com/motosharpley)</p> | <p class="txt-cntr">![John Kwening](https://avatars3.githubusercontent.com/u/4764968?s=60&v=1) [John Kwening](https://github.com/jkwening)</p> | <p class="txt-cntr">![Travis](https://avatars1.githubusercontent.com/u/6627101?s=60&v=1) [Travis](https://github.com/mallek)</p> | <p class="txt-cntr">![tanyagupta](https://avatars1.githubusercontent.com/u/8497274?s=60&v=1) [tanyagupta](https://github.com/tanyagupta)</p> | <p class="txt-cntr">![Sameer Khan](https://avatars1.githubusercontent.com/u/21181432?s=60&v=1) [Sameer Khan](https://github.com/sameerkhan116)</p> | <p class="txt-cntr">![Christopher](https://avatars3.githubusercontent.com/u/21190284?s=60&v=1)  [Christopher Gates](https://github.com/tophergates)</p>
```

Comment:
just a small typo at the top <p class="txt-cntr">![Brayan] <---extra a should be Bryan

Reply:
Oh Sorry about that, fixed it.

Reply:
There is a people link linking to the contributor's page in the github, should I replace that link or add a new link to CONTRIBUTORS.html